draft-ietf-rtcweb-jsep.xml

```diff
@@ -1568,6 +1582,37 @@ Suggested:
           remote candidate, connectivity checks will be sent to the new
           candidate.</t>
         </section>
+        <section anchor="sec.onicecandidate" numbered="true" toc="default">
+          <name>onicecandidate Event</name>
+          <t>The onicecandidate event is dispatched to the application in two 
+          situations: 1) when the ICE agent has discovered a new allowed local
+          ICE candidate during ICE gathering, as outlined in
+          <xref target="sec.ice-gather-overview" format="default"></xref> and 
+          subject to the restrictions discussed in 
+          <xref target="sec.ice-candidate-policy" format="default"></xref>, and
+          2) when an ICE gathering phase completes. The event contains a single
+          IceCandidate object, as defined in 
+          <xref target="sec.ice-candidate-format" format="default"/>.
+          </t>
+          <t>For the first case, the newly discovered candidate is reflected
```

Comment:
I might say "In the first case" but that's style.

Reply:
sg